support/cas-server-support-aws-cognito-authentication/src/main/java/org/apereo/cas/authentication/AmazonCognitoAuthenticationAuthenticationHandler.java

```diff
@@ -42,6 +43,8 @@
 
     private final AmazonCognitoAuthenticationProperties properties;
 
+    private Map<String, String> mappedAttributes = new LinkedHashMap<>();
```

Comment:
Remove. Use the properties instead.

Reply:
I think I need to keep the map constructed from properties here. I will update map name (mappedAttributes) to reflect to use of this map.

Reply:
Replacing inner class with simple map makes this unnecessary. I will remove it.